prow/plank/controller.go

```diff
@@ -200,9 +200,17 @@ func (c *Controller) Sync() error {
 	var syncErrs []error
 
 	pjs := &prowapi.ProwJobList{}
+	timeoutSecs := (*int64)(nil)
+	if dur := c.config().Plank.ProwJobListTimeout; dur != nil {
```

Comment:
Maybe just change to also use a cache-backed client so we reduce the total  number of lists calls needed?

Reply:
Done.